pages/workflow/plan.mdx

```diff
@@ -0,0 +1,105 @@
+---
+layout: 'guide'
+title: '1. Plan analytics update in Avo branch'
+abstract: ''
+---
+
+import Link from '../../components/Link';
+import styles from '../index.module.scss';
+import PageLink, { CallToAction } from '../../components/PageLink';
+import AnalyticsManagerIcon from '../../images/svg/analyticsmanager.svg';
+import DevIcon from '../../images/svg/dev.svg';
+
+import { FunctionComponent } from 'react';
+import Zoom from 'react-medium-image-zoom';
+
+## 1.1 Decide what to track
+
+_Role: Product or Data (This initiative is usually led by a product oriented person)_
+
+The hardest part in the whole analytics release process is usually to decide what events and properties you need, to be able to measure the success of the product change. There are many paths to a decision, but no matter which one you take, we recommend doing it as early as possible in the development process.
+
+We always recommend starting by formulating the metrics you hope to move with your change. Our favorite way to do this is what we like to call a <Link passHref href="/blog/tracking-the-right-product-metrics"><a>“Purpose Meeting”</a></Link>. The outcomes of that meeting are:
+1. Goals – what does success look like?
+2. Metrics – how will we measure how successful we are?
+3. Data – what analytics events do we need for our success metrics?
+
+The result of the purpose meeting is usually a draft of events that someone takes forward to the next step.
+
+ <div className={styles.row}>
+          <PageLink
+            image={AnalyticsManagerIcon}
+            title="Purpose Meeting"
```

Comment:
I think it's good to set the stage that this takes you out of the docs and into the blog
```suggestion
            title="blog Post: Purpose Meeting"
```